airdrop-claim-reject-cancel-hts-token-js-sdk/utils/tokenOperations.js

```diff
@@ -0,0 +1,66 @@
+import { TokenCreateTransaction, TokenMintTransaction, TransferTransaction } from "@hashgraph/sdk";
+import * as queries from "./queries.js";
+
+export async function createHtsTokenFcn(tkName, tkSymbol, trId, tkType, sType, iSupply, maxSupply, listOfKeys, trPvKey, client) {
+	const tokenCreateTx = await new TokenCreateTransaction()
+		.setTokenName(tkName)
+		.setTokenSymbol(tkSymbol)
+		.setTreasuryAccountId(trId)
+		.setTokenType(tkType)
+		.setSupplyType(sType)
+		.setDecimals(0)
+		.setInitialSupply(iSupply)
+		.setMaxSupply(maxSupply)
+		// .setCustomFees([customFee])
+		.setAdminKey(listOfKeys[0].publicKey)
+		.setSupplyKey(listOfKeys[1].publicKey)
+		.setPauseKey(listOfKeys[2].publicKey)
+		.setFreezeKey(listOfKeys[3].publicKey)
+		.setWipeKey(listOfKeys[4].publicKey)
+		// .setKycKey(listOfKeys[5].publicKey)
```

Comment:
Don't need discard

Reply:
Leaving the KYC Key line commented for teaching/learning purposes.
This key is not passed to the util function as enabling KYC requires approval before transferring the token.